### DISCUSSION TITLE
Correct Apache2.conf for UI questions

### DISCUSSION THREAD

**Discussion by gsg-git** [osm-search/Nominatim#3220] [Q&A]
Hello my nominatim is running and the WEB search requestst is running.
But i have trouble gettiing the UI running:
The nominatim UI website is up but it cant connect to the files in http://serverip/nominatim/status.php for example.
Searches fails too.
Error fetching data from http://localhost/nominatim/search.php?q=berlin&polygon_geojson=1&format=jsonv2 (TypeError: Failed to fetch)
The apace settings:
<Directory "/nominatim/nominatim-ui/dist">
  DirectoryIndex search.html
  Require all granted
</Directory>

Alias /nominatim/ui /nominatim/nominatim-ui/dist

Alias /nominatim /nominatim/website

<Directory "/nominatim/website">
  Options FollowSymLinks MultiViews
  AddType text/html   .php
  Require all granted

  RewriteEngine On

  # This must correspond to the URL where nominatim can be found.
  RewriteBase "/nominatim/"

  # If no endpoint is given, then use search.
  RewriteRule ^(/|$)   "search.php"

  # If format-html is explicitly requested, forward to the UI.
  RewriteCond %{QUERY_STRING} "format=html"
  RewriteRule ^([^/]+)(.php)? ui/$1.html [R,END]

  # If no format parameter is there then forward anything
  # but /reverse and /lookup to the UI.
  RewriteCond %{QUERY_STRING} "!format="
  RewriteCond %{REQUEST_URI}  "!/lookup"
  RewriteCond %{REQUEST_URI}  "!/reverse"
  RewriteRule ^([^/]+)(.php)? ui/$1.html [R,END]
</Directory>

Than you for every little help!

**Comment by mtmail** [comment]
duplicate of https://github.com/osm-search/nominatim-ui/issues/231
